INSTALLATION.md

```
@@ -380,3 +390,84 @@
+   ```bash
+   sudo useradd -r -s /bin/false talawa_admin
+   sudo groupadd -r talawa_admin
+   sudo usermod -a -G talawa_admin talawa_admin
+   ```
+
+---
+
+### Service Configuration
+
+#### 1. **Copy the `talawa_admin.service` file**
+   - Place the `talawa_admin.service` file in the appropriate systemd directory based on your Linux distribution:
```

Markdownlint (0.37.0)

Expected: 0; Actual: 3
Unordered list indentation

(MD007, ul-indent)